code/socialDistribution/requests.py

```diff
@@ -0,0 +1,54 @@
+""" File that contains wrapper methods for all API HTTP requests. Use this file for makeing GET, POST, PUT, 
+    DELETE, etc requests to any social distribution API server. Functions in this file just act as a wrapper for 
+    adding appropriate headers and parsing the response as JSON.
+"""
+
+import requests
+
+
+def get(url, params=None):
+    """ Makes a GET request at the given URL and returns the JSON body of the HTTP response. 
+
+        Parameters:
+         - url (string): The URL endpoint for the HTTP request
+         - params (dict): The query string parameters (default is None)
+
+        Returns:
+         - (dict): JSON response data if status code of request is 200 OK. Otherwise, return None
+    """
+
+    headers = {
+        "Accept": "application/json"
+    }
+
+    response = requests.get(url, headers=headers, params=params)
+
+    if response.status_code == 200:
+        return response.json()
+    else:
+        return None
+
+
+def post(url, params=None, body={}):
```

Comment:
I might have to modify this to accept headers from outside in my pr.

Reply:
Could we put the headers inside the function definition themselves? That's what I was thinking initially.